tmpl/src/{{module_name}}/cli.py.j2

```diff
@@ -2,6 +2,9 @@
 # Third-party
```

Comment:
Need to import `logging`.

Reply:
Thanks, Stefan, and sorry to bother you with my errors. The review request of the PR gets out before the github actions with all the tests have finished. How can we avoid this?

Reply:
I will do the fixes until all tests are successful.

Reply:
Ah, so I was just too fast... :) I don't know whether there's a more proper way to delay notifications, but you can just skip the reviewers when opening the PR and assign them retroactively once the tests validate.